CHANGELOG.md

```diff
@@ -1,5 +1,13 @@
 # CHANGELOG
 
+## Next Release
+
+* Add getter and setter for eel_pfc in CustomsInfo class
+* Add getter and setter for id in Brand class
+* Fix bug for `create()` method in Refund class
+* Add test suite and upgrade JUnit4 to JUnit5
+* Add assertj dependency to compare two objects in unit test
```

Comment:
This line isn't necessary to callout since it doesn't effect the user. The CHANGELOG is typically reserved for big highlights that change behavior of the app, but normal users will never run this test suite.